README.md

```diff
@@ -53,6 +53,7 @@ reported on the mailing list nonetheless.
 
 Dependencies
 ------------
+> **Note**: Building bpftool is only supported on Linux. The build process relies on Linux-specific headers and tools, and is not expected to work on macOS or other non-Linux systems.
```

Comment:
The syntax for callouts for GitHub-rendered Markdown is the following. Please also leave a blank line under the heading.

```suggestion

> [!NOTE]
> Building bpftool is only supported on Linux. The build process relies on
> Linux-specific headers and tools, and is not expected to work on macOS or
> other non-Linux systems.
```